File: Day 2/arithmetic.py
original_price = float(raw_input())
tip = int(raw_input())
tax = int(raw_input())
final_price = original_price + float(original_price * tip) / 100. + float(original_price * tax) / 100.

print 'The final price of the meal is $%d.' % (int(round(final_price)))

File: Day 2/arithmetic.scala
object Solution {

    def main(args: Array[String]) {
        var originalPrice : Float = scala.io.StdIn.readFloat;
        var tip : Int = scala.io.StdIn.readInt;
        var tax : Int = scala.io.StdIn.readInt;
        var res : Float = originalPrice * (1 + tip.toFloat / 100 + tax.toFloat / 100);
        var finalPrice : Int = scala.math.round(res);
        println("The final price of the meal is $" + finalPrice + ".");
    }
}